src/app/shared/components/button/button.component.html

```diff
@@ -0,0 +1,22 @@
+<button
+  (click)="onClick()"
+  [disabled]="isDisabled"
+  [class]="btnClass"
+  [ngStyle]="{ 'border-color': borderColor }"
```

Comment:
Do you need it?

Reply:
44d608c1e184d7ef5d4fd6d12aef80d4809997b5